governance.md

```diff
@@ -326,15 +326,26 @@ Partners, with associated benefits:
 Changes to the governance documents are submitted via a GitHub pull
 request to The Project's governance documents GitHub repository at
 [https://github.com/jupyter/governance](https://github.com/jupyter/governance).
-The pull request is then refined in response to public comment and
-review, with the goal being consensus in the community.  After this
-open period, a Steering Council Member proposes to the Steering
-Council that the changes be ratified and the pull request merged
-(accepting the proposed changes) or proposes that the pull request be
-closed without merging (rejecting the proposed changes).  The Member
-should state the final commit hash in the pull request being proposed
-for acceptance or rejection and briefly summarize the pull request. All votes
-are limited in time to 4 weeks after the vote is initiated. At the end of 4
+There are two phases to the voting process:
+
+**The discussion phase** begins when the person first opens a pull-request. During
+this time, *the pull-request must be in a draft state*.
+The pull request is refined in response to public comment and
+review, with the goal being consensus in the community.
+
+The pull request author may *call a vote* when they believe enough feedback and
+iteration has occurred. This is triggered by moving the pull request from the
+*draft state* to an *active state*. This triggers the voting phase.
+
+**The voting phase** begins when the PR enters an active state. The proposed changes
+in the pull request are frozen, and may not be substantively modified after voting
+has begun. If the author (or a steering council member) wishes to substantively change
+the proposed changes, they should close the pull request and re-start the process
+with a new pull request. During the voting phase, the Steering Council
```

Comment:
This indicates that the author or a steering council can preemptively decide in the negative (i.e., effect the same change as a negative vote). I think standard parliamentary procedure is that once the voting starts (and I think even when the discussion starts), the proposal belongs to the group then, not the author, and must run the course of voting without being preemptively withdrawn.

I would delete the sentence about the author being able to close the PR, and the voting should run its course.
```suggestion
has begun. During the voting phase, the Steering Council
```

Reply:
> I would delete the sentence about the author being able to close the PR, and the voting should run its course.

In other words, I think there should be consensus that the current proposal should be abandoned, and at the voting phase, consensus is expressed through votes.